Javascript/q44.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>q44</title>
+
+  <style>
+    body {
+      font-family: Verdana, Geneva, Tahoma, sans-serif;
+    }
+
+    th {
+      text-align: left;
+    }
+  </style>
+
+  <script>
+    function myFunction() {
+      const length = document.querySelector("#demo0").value;
+      const count = document.querySelector("#demo1").value;
+      const charList = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789@#$%&*-_';
+      let x =' ';
+      for (let j=0; j<count; j++){
+        for(let i=0; i<length; i++) {
+          const num = Math.round(Math.random() * charList.length);
+          x = x + charList.substring(num, num+1);
```

Comment:
There are more efficient ways of getting a single character at a given index in a string than `substring`. `substring` is good if you want to get a multi-character string out of the original string.

Reply:
done